### ISSUE TITLE
[bug] v1.44.0 for windows complain about "ModuleNotFoundError: No module named 'distutils.dir_util'"

### ISSUE THREAD

**Issue by nowgain-nowpain** [conan-io/conan#10318]
### Environment Details (include every applicable attribute)
  * Operating System+version: Windows 10
  * Compiler+version: msvc 14
  * Conan version: 1.44.0
  * Python version: no (Don't have any version of python installed in the system)

### Steps to reproduce (Include if Applicable)
1) download  conan v1.44.0 x86_64 installer for windows from (https://conan.io/downloads.html as of (Jan112022).
2) Install conan v1.44.0 with the installer and using all defaults.
3) for a customer package with customized conan recipe, which imports  distutils.dir_util  at the top of the recipe:
```
from conans import ConanFile, CMake, tools
from distutils.dir_util import copy_tree
from shutil import copyfile

import platform, os

class MyConan(ConanFile):

```
When run conan install, conan v1.44.0 complains:

MyConan/0.11.0@user/stable: Not found in local cache, looking in remotes...
MyConan/0.11.0@user/stable: Trying with 'conan'...
Downloading conanmanifest.txt completed [0.28k]
Downloading conanfile.py completed [3.06k]
MyConan/0.11.0@user/stable: Downloaded recipe revision 98b2550de0ebca6d166e103b4ce9b502
ERROR: MyConan/0.11.0@user/stable: Cannot load recipe.
Error loading conanfile at 'C:\Users\my_user_name\.conan\data\MyConan\0.11.0\user\stable\export\conanfile.py': Unable to load conanfile in C:\Users\my_user_name\.conan\data\MyConan\0.11.0\user\stable\export\conanfile.py
  File "<frozen importlib._bootstrap>", line 684, in _load
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "C:\Users\my_user_name\.conan\data\MyConan\0.11.0\user\stable\export\conanfile.py", line 2, in <module>
    from distutils.dir_util import copy_tree
ModuleNotFoundError: No module named 'distutils.dir_util'

4) uninstall v1.44.0, download and install v1.43.0 or other previous versions. It works without the issue.

Seems like the distutils module is not installed for v1.44? 
Thanks.

**Comment by memsharded** [comment]
Managed to reproduce this, and submitted a potential fix for 1.44.1: https://github.com/conan-io/conan/pull/10320

Seems the original cause is some changes in the python virtualenv package. Thanks for reporting!

As general feedback, this is one of the issues of the windows self-contained installer, it is a bit more fragile regarding the python interpreter and python stdlib, that might not have all the desired modules (only those used by Conan in that specific release).

**Comment by memsharded** [comment]
Fixed in https://github.com/conan-io/conan/pull/10320, will be in 1.44.1 soon
